src/app/zap-templates/templates/app/cluster-objects.zapt

```diff
@@ -25,6 +25,13 @@ namespace Clusters {
 namespace detail {
 // Structs shared across multiple clusters.
 namespace Structs {
+
+{{#zcl_enums}}
+{{#if has_more_than_one_cluster}}
+using {{asUpperCamelCase name}} = Clusters::detail::Enums::{{asUpperCamelCase name}};
```

Comment:
Why do we need to import the enum names into the _Structs_ namespace?  That does not look right.

The real issue is that we set up our structure differently for namespace detail and for clusters, right?  In clusters, enums are not inside an Enums namespace....  I would say we should not put them under one under `Clusters::detail` either.

Reply:
But this is not adding it "where it used to be".  This is adding a `using` for every struct that uses the enum, no?